src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
you shouldn't allow creating an instance of HibernateUtil several times

Reply:
fixed

Reply:
```suggestion
    private HibernateUtil() {
```